### PR TITLE
Fix: Contact Buttons - various fixes

### PR DESCRIPTION
**Pull request by mserino**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/conversion-center/assets/scss/widgets/contact-buttons-editor.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
```
Comment (louiswol94) [inline modules/conversion-center/assets/scss/widgets/contact-buttons-editor.scss:1]: This should be placed in `modules/conversion-center/assets/scss/editor/contact-buttons-editor.scss`

File: modules/conversion-center/assets/scss/widgets/contact-buttons-vars.scss [hunk 2]
```diff
@@ -33,4 +33,4 @@ $contact-buttons-animation-delay: 0;
 
 $contact-buttons-chat-box-bg: #FFFFFF;
 
-$contact-buttons-contact-gap: 15px;
\ No newline at end of file
+$contact-buttons-contact-gap: 15px;
```
Comment (louiswol94) [inline modules/conversion-center/assets/scss/widgets/contact-buttons-vars.scss:36]: Missing empty line

File: modules/conversion-center/assets/scss/widgets/contact-buttons.scss [hunk 3]
```diff
@@ -232,6 +242,27 @@
...
+		--e-contact-buttons-message-bubble-body: #000000;
+		--e-contact-buttons-message-bubble-time: #000000;
+		--e-contact-buttons-message-bubble-name: #000000;
+		--e-contact-buttons-message-bubble-bubble-bg: #ffffff;
+		--e-contact-buttons-message-bubble-chat-bg: #ECE5DD;
+		--e-contact-buttons-send-button-icon: #ffffff;
+		--e-contact-buttons-send-button-bg: #33CCFF;
+		--e-contact-buttons-send-button-icon-hover: #ffffff;
+		--e-contact-buttons-send-button-bg-hover: #0099FF;
+		--e-contact-buttons-chat-box-bg: #FFFFFF;
```
Comment (louiswol94) [inline modules/conversion-center/assets/scss/widgets/contact-buttons.scss:263]: Can you double check if we have system variables for some of these colours?

**Review by louiswol94 — CHANGES_REQUESTED** [review]

File: modules/conversion-center/assets/scss/widgets/contact-buttons-vars.scss (hunk 2 above)
Comment (mserino) [reply modules/conversion-center/assets/scss/widgets/contact-buttons-vars.scss:36]: 🤔 I can see the empty line here

File: modules/conversion-center/assets/scss/widgets/contact-buttons.scss (hunk 3 above)
Comment (mserino) [reply modules/conversion-center/assets/scss/widgets/contact-buttons.scss:263]: I have found some of them but just for the main color, not for the hover background or text, and not for all social platforms.

File: modules/conversion-center/assets/scss/frontend.scss [hunk 4]
```diff
@@ -13,3 +13,5 @@
 @import "widgets/contact-buttons-var-6";
 @import "widgets/contact-buttons-var-7";
 @import "widgets/contact-buttons-var-8";
+
+@import "editor/contact-buttons-editor.scss";
```
Comment (louiswol94) [inline modules/conversion-center/assets/scss/frontend.scss:17]: @mserino I think the editor imports are here: `elementor/assets/dev/scss/editor/editor.scss`
We don't need this SCSS in the frontend file.

Comment (mserino) [reply modules/conversion-center/assets/scss/frontend.scss:17]: ah, thanks!

Comment (mserino) [reply modules/conversion-center/assets/scss/frontend.scss]: @louiswol94 okay, pushed a fix for this. It needed to be imported in the `preview`

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**